rskj-core/src/main/java/co/rsk/trie/Trie.java

```diff
@@ -151,7 +154,10 @@ public static Trie fromMessage(byte[] message, TrieStore store) {
             trie = fromMessageRskip107(ByteBuffer.wrap(message), store);
         }
 
+        trie.saved = true;
```

Comment:
Isn't this `saved` redundant as it is gonna pass from either `frommessacheOrchid` or `fromMessageRskip107` which already mark the trie as `saved`?